test/jdk/tools/jpackage/clean_test_output.sh

```diff
@@ -76,6 +79,41 @@ filterFile () {
 
       # Convert variable part of stack trace entry `at jdk.jpackage.test.JPackageCommand.execute(JPackageCommand.java:863)`
       -e 's/^\(.*\b\.java:\)[0-9]\{1,\}\()\r\{0,1\}\)$/\1N\2/'
+
+      # Whipe out entire output of /usr/bin/hdiutil command.
+      # It is of little to no interest and contains too many variable parts to deal with individually.
+      -e '/^Running \/usr\/bin\/hdiutil/,/^Returned:/{
+            //,/^Output:/!d
+          }'
+
+      # Zip stack traces.
+      -e $'/^\tat /{
+            :a
+            g
+            N
+            s/.*\\n//
+            /^\tat /ba
+            s/\\(^\t... \\)[0-9]\\{1,\\}\\( more\\)/\\1N\\2/
+            s/\(.*\)/\tat <stacktrace>\\n\\1/
+            P
+            D
+          }'
+
+      # Convert PID value in `taskkill /F /PID 5640`
+      -e 's|taskkill /F /PID [0-9]\{1,\}|taskkill /F /PID <pid>|'
+
+      # Convert PID value in `The process with PID 5640 has been terminated`
```

Comment:
I am confused with this comment. Should we use something more generic then 5640. Same below for 57182ms.